src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,2952 @@
+            yield return new object[] { (LongEnum)long.MaxValue };
+            yield return new object[] { float.MinValue };
+            yield return new object[] { (float)(-1) };
+            yield return new object[] { float.MaxValue };
+            yield return new object[] { float.PositiveInfinity };
+            yield return new object[] { float.NegativeInfinity };
+            yield return new object[] { float.NaN };
+            yield return new object[] { double.MinValue };
+            yield return new object[] { (double)(-1) };
+            yield return new object[] { double.MaxValue };
+            yield return new object[] { double.PositiveInfinity };
+            yield return new object[] { double.NegativeInfinity };
+            yield return new object[] { double.NaN };
+            yield return new object[] { decimal.MinValue };
+            yield return new object[] { (decimal)(-1) };
+            yield return new object[] { decimal.MaxValue };
+            yield return new object[] { "-1" };
+            yield return new object[] { "18446744073709551616" };
+            yield return new object[] { "1844674407370955161618446744073709551616" };
+            yield return new object[] { double.PositiveInfinity.ToString() };
+            yield return new object[] { double.NegativeInfinity.ToString() };
+            yield return new object[] { double.NaN.ToString() };
+        }
+
+        [Theory]
+        [MemberData(nameof(ToByte_OverflowObject_TestData))]
+        public void ToByte_OverflowObject_ThrowsOverflowException(object value)
+        {
+            Assert.Throws<OverflowException>(() => Conversions.ToByte(value));
+        }
+
+        public static IEnumerable<object[]> ToSByte_String_TestData()
+        {
+            yield return new object[] { null, (sbyte)0 };
+            yield return new object[] { "-1", (sbyte)(-1) };
+            yield return new object[] { "0", (sbyte)0 };
+            yield return new object[] { "1", (sbyte)1 };
+            yield return new object[] { "&h5", (sbyte)5 };
+            yield return new object[] { "&h0", (sbyte)0 };
+            yield return new object[] { "&o5", (sbyte)5 };
+            yield return new object[] { " &o5", (sbyte)5 };
+            yield return new object[] { "&o0", (sbyte)0 };
+            yield return new object[] { 1.1.ToString(), (sbyte)1 };
+        }
+
+        [Theory]
+        [MemberData(nameof(ToSByte_String_TestData))]
+        public void ToSByte_String_ReturnsExpected(string value, sbyte expected)
+        {
+            AssertEqual(expected, Conversions.ToSByte(value));
+        }
+
+        [Theory]
+        [MemberData(nameof(InvalidString_TestData))]
+        public void ToSByte_InvalidString_ThrowsInvalidCastException(string value)
+        {
+            Assert.Throws<InvalidCastException>(() => Conversions.ToSByte(value));
+        }
+
+        public static IEnumerable<object[]> ToSByte_OverflowString_TestData()
```

Comment:
remove this and use this
MemberData(nameof(ToByte_OverflowString_TestData)] instead.
Maybe change the name of the function

Reply:
Yes I've consolidated the test data